scripts/data_functions.R

```diff
@@ -104,6 +104,9 @@ check_data <- function(sd, des, con){
     stopifnot(all(des$original_names %in% colnames(sd)))
     # Sanity check: each column in the count data should have a corresponding sample (row) in the metadata
     stopifnot(all(colnames(sd) %in% des$original_names))
+    # Sanity check: config['group_facet'] not an element in config['intgroup_to_plot'] 
+    if(params$group_facet %in% params$intgroup_to_plot)
+        stop("The column for faceting should not be an element in your intgroup_to_plot list.")
```

Comment:
This is good, but I wonder if it would fit better in `check_required_params()`? The main reason would be to consolidate similar checks. I think it may also be called earlier in the workflow but maybe you can double check that.

Reply:
Good idea. I've done this and added a commit for it.

I've tested this out locally with the test dataset.